contributors/chairs-and-techleads/process-workflow-guide.md

```diff
@@ -0,0 +1,250 @@
+  - [Example: KEP-Specific Issue Triage](#example-kep-specific-issue-triage-sig-release) (SIG Release)
+- [KEP Planning for an Upcoming Release Cycle](#kep-planning-for-an-upcoming-release-cycle) (Network, Storage)
+  - [Example: Prioritization Framework](#example-prioritization-framework-sig-release) (SIG Release)
+  - [Example: KEP Tracking Table/Google Doc](#example-kep-tracking-tablegoogle-doc-sigs-node-windows) (SIGs Node, Windows)
+  - [Example: KEP Tracking Spreadsheet](#example-kep-tracking-spreadsheet-sig-storage) (SIG Storage)
+  - [Tips for Handling KEPs That Require Cross-SIG Coordination](#tips-for-handling-keps-that-require-cross-sig-coordination)
+  - [Managing KEP Scope](#managing-kep-scope)
+  - [When You Need More Help with KEPs/KEP Process](#when-you-need-more-help-with-kepskep-process)
+- [Creating Roadmaps](#creating-roadmaps)
+  - [Example: Roadmapping Templates for a Release Cycle](#example-roadmapping-templates-for-a-release-cycle-sigs-docs-release) (SIGs Docs, Release)
+  - [Example: Brainstorming Template for Creating a Roadmap](#example-brainstorming-template-for-creating-a-roadmap-sig-releaserelease-engineering) (SIG Release/Release Engineering)
+  - [Example: Umbrella Issues for Roadmapping an Initiative/Epic](#example-umbrella-issues-for-roadmapping-an-initiativeepic-sigs-testing-release) (SIGs Testing, Release)
+- [Managing Communications: Delegating and Driving Transparency](#managing-communications-delegating-and-driving-transparency)
+  - [Slack](#slack)
+  - [Mailing List and GitHub Notification Best Practices](#mailing-list-and-github-notification-best-practices) (SIG Contributor Experience)
+
+For brevity’s sake, not every aspect of running a SIG, WG, or subproject is covered. If you have questions, please reach out in [#chairs-and-techleads](https://kubernetes.slack.com/messages/chairs-and-techleads).
+
+## Ways of Working Agreement
+
+This one comes first as it’s a helpful way to summarize how your SIG, WG, or subproject operates in terms of communication, processes, and roles and responsibilities. This can serve as a useful onboarding tool or help you prevent process drift or abandonment. 
+
+### Example: SIG Release/Release Engineering
+
+The [Release Engineering](https://github.com/kubernetes/sig-release/tree/master/release-engineering) subproject of SIG Release has published a Ways of Working agreement that covers communication channels and how they’re used; roles and responsibilities; how they discuss work; and other major aspects of workflow and process. 
+
+### Making the Most out of Meetings
+
+Most, if not all, SIGs, WGs, and subprojects already have templates for organizing bimonthly meetings. [Here is a collection of templates](https://miro.com/templates/team-meetings/) for remote meetings that you might also find useful.
+
+You’ll find some SIG-specific meeting tips in Release Engineering’s WoW agreement. A few more:
```

Comment:
What is "Release Engineering’s WoW agreement"? Is there a link?

Reply:
There's a link earlier in the guide -- WoW = "Ways of Working"

Reply:
oh, that makes sense. I didn't connect the abbreviation to the previous text. sorry

Reply:
No problem :)